emcc.py

```diff
@@ -3313,9 +3307,16 @@ def is_int(s):
     return False
 
 
+def main(args):
+  start_time = time.time()
+  rtn = run(args)
```

Comment:
how about `ret` that is used in other places?

Reply:
This is just a normal function-local variable isn't it.  I don't think we have any globals with this name.  At least we certainly shoudln't.

Reply:
Sorry I misread.  Will fix.